src/server/scripts/Commands/cs_deserter.cpp

```diff
@@ -106,17 +107,17 @@ class deserter_commandscript : public CommandScript
             return false;
         }
 
-        if (time.empty())
+        if (!time)
         {
-            return false;
+            time = isInstance ? "30m" : "15m";
         }
 
         Player* target = player->GetConnectedPlayer();
-        int32 duration = TimeStringToSecs(time);
+        int32 duration = TimeStringToSecs(time.value());
 
         if (duration == 0)
         {
-            duration = atoi(time.c_str());
+            duration = atoi(time.value().c_str());
```

Comment:
don't use `atoi` we have `Acore::StringTo`

Reply:
Oh didn't know that. Thanks :)
I'll change it later